python/tvm/relay/backend/_backend.py

```diff
@@ -90,7 +90,11 @@ def _tensor_value_repr(tvalue):
 
 @tvm._ffi.register_func("relay._constant_repr")
 def _tensor_constant_repr(tvalue):
-    return str(tvalue.data.asnumpy())
+    # TODO(gus) do this in a smarter way
+    try:
```

Comment:
Here, we should actually check for a custom type (`tvm.datatype.get_type_registered()`). Using try/except was my lazy solution. So something like:
```
if tvm.datatype.get_type_registered(tvalue.data.dtype.code):
  ...
else:
  return str(tvalue.data.asnumpy())
```

Reply:
yep done